### DISCUSSION TITLE
How to try the examples?

### DISCUSSION THREAD

**Discussion by luxlux** [vuejs/petite-vue#201] [Q&A]
I never tried vue or petite-vue before.

I wanted to check if it is useful for me. 
i have downloaded into my webfolder on my v-server using `git clone https://github.com/vuejs/petite-vue`

Then i opened the petite-vue folder in the browser. It displays index.html with the examples.
But all links do not work. They are all not relative links like: `<a href="/examples/todomvc.html">TodoMVC</a>`
I removed the first "/" to get the example opened on it's relative folder.

The example file now loads, but on console i get an error: 
`GET https://digiscales.de/test/petite-vue/src/ 403 (Forbidden)` on todomvc.html:12
 on line 12 there is this code:
` import { createApp } from '../src'`

Does it work to load a remote folder instead of a file? The standard setting on webservers is to not return the content of folders.
Anyhow, if i check the folder /src there are no .js files. Only .ts files. Sounds like typescript (which i never used before. is this not something which have to be rendered before use?)

Unfortunately there is no info given how to install the examples! Not really useful to impress new users... ;-)
So what i have to do to get at least the examples to work?
BTW: Should petite-vue not simply run, with including the petite-vue script to the html?

I still tried something like `npm install` and `npm build`. The first did something, the second not.
But the examples are still dead...

**Comment by sqllyw** [comment]
One way is:

1) expand the petite-vue-main.zip to a folder, cd into that folder
2) npm install vite
3) npm run dev
4) open http://localhost:3000/ in a browser

you will see Examples and Tests
